lisa/platforms/platinfo.py

```diff
@@ -108,6 +115,8 @@ def add_target_src(self, target, rta_calib_res_dir, src='target', **kwargs):
         if target.is_module_available('sched'):
             info['cpu-capacities'] = target.sched.get_capacities(default=1024)
 
+        info['kernel']['symbols-address'] = self._read_kallsyms(target)
```

Comment:
Should this be a deferred value?

Reply:
there isn't really a good reason for a deferred value here, that's fast to get. Deferred values have a number of drawbacks, and that piece of info will be necessary if we want to plot anything offline.